### PR TITLE
Unit Dropdown for Curriculum Quick Assign

### PR DESCRIPTION
**Pull request by hannahbergam**
The unit dropdown now adds a unit to the course object in the section. It also adds tts and lesson extras params if they exist (but leaves as undefined if no unit is selected). 

Screenshot 2023-03-27 at 4 01 57 PM
Screenshot 2023-03-27 at 4 02 07 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/TEACH-327
- https://codedotorg.atlassian.net/browse/TEACH-367

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx [hunk 1]
```diff
@@ -37,33 +66,25 @@ export default function VersionUnitDropdowns({
...
-                    <option key={courseOffering.id} value={courseOffering.id}>
-                      {courseOffering.display_name}
-                    </option>
-                  ))}
-                </optgroup>
-              ))}
-            </select>
-          </span>
-        )}
+        {sectionCourse?.versionId !== 0 &&
```
Comment (bethanyaconnor) [inline apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx]: Why 0?

**Review by bethanyaconnor — APPROVED** [review]
One small question but otherwise looks great!!

File: apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx (hunk 1 above)
Comment (hannahbergam) [reply apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx]: Excellent question... I copied this logic from the existing component so I'm not sure but I'll go look!

Comment (hannahbergam) [reply apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx]: Fwiw- I couldn't find any explicit documentation around this check. I don't think we ever use 0 as a default value... should I replace with 'sectionCourse?.versionId &&' or just leave it as-is and trust that the existing logic was built that way for a reason?

Comment (bethanyaconnor) [reply apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx]: I personally would prefer something like `sectionCourse?.versionId &&`. 0 is falsey so it should maintain the existing behavior anyway

Comment (hannahbergam) [reply apps/src/templates/sectionsRefresh/VersionUnitDropdowns.jsx]: Updated! I also caught and fixed a small bug. Now selecting a new version clears out an old unit assignment.

**Pull request merged**